<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="callBtn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>40</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>call</string>
   </property>
  </widget>
  <widget class="QPushButton" name="talkBtn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>140</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>talk</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tokenBtn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>220</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>token</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
